server/game/cards/11.4-MoD/MeeraReed.js

```diff
@@ -0,0 +1,41 @@
+const DrawCard = require('../../drawcard');
+
+class MeeraReed extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onSacrificed: event => this.isControlledStarkCharacter(event.cardStateWhenSacrificed)
+            },
+            handler: context => {
+                this.game.addMessage('{0} uses {1} to return {1} to shadows', context.player, this);
+                context.player.putIntoShadows(this, false);
```

Comment:
I struggle to think of a reason why, but I think you should be able to save versus this if you want. Unless this is on purpose to diminish (likely useless) prompts, which is fine with me.

Reply:
The closest scenario I can think of is if you duped Meera, then the opponent steals Meera, and tries to trigger this ability. You might (maybe??) want to save in that scenario. But since she would be returned to your shadows, the opponent probably doesn't want to do that in the first place.

Mostly the cannot save here is to prevent automatic dupes from keeping Meera in play after the player explicitly triggers an ability to get her out of play.